dist/ion-gallery.js

```diff
@@ -42,6 +43,8 @@
           }
         });
       }());
+
+      $scope.customCallback = typeof $scope.ionItemCallback !== 'undefined';
```

Comment:
great question... as I want doing trial and error I did try to use the angular.isFunction to test. But then I read that the directive return a NOOP function even if the attribute is not set. But that was before I realized I needed the ? symbol to make the attribute optional. It would probably work with that set.